### ISSUE TITLE
Processing icons uses manifest_url when it should probably use start_url

### ISSUE THREAD

**Issue by dmurph** [w3c/manifest#1004]
https://www.w3.org/TR/appmanifest/#processing-image-resources

This uses the manifest_url as the base url, when this can be a different origin from the start_url origin (and scope). This seems like maybe something that should be fixed?

**Comment by dominickng** [comment]
Icons can often live on a CDN that is a totally different origin from the start_url / scope. Is there a strong reason to require icons to live on the same origin as the start URL?

**Comment by marcoscaceres** [comment]
Agree. The scope is only for navigation, not for resource loading, which is governed by CSP.
